source/patterns/@aws-solutions-constructs/core/lib/cloudfront-distribution-helper.ts

```diff
@@ -215,4 +178,14 @@ export function CloudFrontOriginAccessIdentity(scope: cdk.Construct, comment?: s
   return new cloudfront.OriginAccessIdentity(scope, 'CloudFrontOriginAccessIdentity', {
     comment: comment ? comment : `access-identity-${cdk.Aws.REGION}-${cdk.Aws.STACK_NAME}`
   });
-}
\ No newline at end of file
+}
+
+function getLoggingBucket(cloudFrontDistributionProps: cloudfront.DistributionProps | any, scope: cdk.Construct): s3.Bucket | undefined {
+  return cloudFrontDistributionProps?.enableLogging !== false
```

Comment:
Parsing this logic and comparing it to the original logic had me doing my homework on truthy, falsey and missing values...

Reply:
hehe, sorry I should have probably explained it in more depth, though I had gotten engulfed with the snapshot updates

Reply:
It's much easier to follow than the original code - my struggles were determining the behavior of the original code and comparing it to this.

Reply:
I guess this is where the extensive test suite pays off! :)